tracer/src/Datadog.Trace/IAST/Helpers/CollectionExtensions.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="CollectionExtensions.cs" company="Datadog">
```

Comment:
Can you most this file from the `IAST` folder to `Iast` so it matches the other files?

Reply:
Git doing Git things... This is like a zombie that comes back from time to time. Solved (I hope this time for good)